usage/usageplot.py

```diff
@@ -1,5 +1,6 @@
 """Create a plot of the lsst-dev node useage over time and prints the total
-node_hours of the jobs provided to usage.py.
+node_hours of the jobs provided to usage.py. Also prints out the elapsed
+time spent for each code.
```

Comment:
When writing docstring for a module/function/class, please:
1. Keep short summaries one-line long. Just put any other information you think maybe helpful for the user in the full description.
2. Do not reference implementation details (here, `node_hours`), focus on code's goal/role, provided functionalities.

Keeping that in mind, I would rewrite the beginning of the docstring as:
```
"""Create a plot of the lsst-dev node usage.

Module creates the plot illustrating node usage of the lsst-dev cluster during an LSST campaign. It also prints campaign's short summary including:
- the total number of node hours,
- the elapsed time for each task (i.e. how much time it took to complete all coadd jobs, etc.)

<docstring reminder>
"""
```